_sources/CSPPracticeQuestions/exam24_3.rst

```diff
@@ -72,10 +72,10 @@ You will not be able to change your answers after you hit the "Finish Exam" butt
        :answer_c: What is the average annual precipitation?
        :answer_d: What is the average daily temperature? 
        :correct: c
-       :feedback_a: Incorrect
-       :feedback_b: Incorrect
-       :feedback_c: Correct
-       :feedback_d: Incorrect
+       :feedback_a: Incorrect. The meteorologists have temperature data from day and nights.
+       :feedback_b: Incorrect. The meteorologists have temperature and precipitation data.
```

Comment:
It would be nice to say something like "The meteorologists have temperature and precipitation data, so they can compare the two and find out if they correlate." Give a little more description.